src/specimens/Html.js

```diff
@@ -108,27 +174,57 @@ class Html extends React.Component {
       ? <div style={styles.toggle} onClick={this.toggleSource.bind(this)}>&lt;&gt;</div>
       : null;
 
+    let responsive = options.device === 'select' || deviceFoundInList === undefined
+      ? <div style={styles.screensizes}>
+          { devices.map( (sc, i) => <div
+            key={i}
+            style={{...styles.screensize, ...(sc.name === screenSize.name ? styles.screensizeActive : null) }}
+            onClick={this.setSize.bind(this, sc)}>
+              {sc.name}
+              {sc.name === screenSize.name
+                ? <small>(@{screenSize.width}x{screenSize.height}{(sc.width > parentWidth) && ' scaled'})</small>
+                : <small>{(sc.width > parentWidth) && '(scaled)'}</small>
+              }
+            </div>
+          )}
+        </div>
+      : <div style={styles.screensizes}>
+          {
+            options.device && screenSize.width > parentWidth
+            ? <div>{options.device} <small>(@{screenSize.width}x{screenSize.height}, scaled)</small></div>
+            : <div>{options.device} <small>(@{screenSize.width}x{screenSize.height})</small></div>
+          }
+        </div>;
+
     const content = <div dangerouslySetInnerHTML={{__html: children}} />;
 
     return (
       <div ref='specimen' style={styles.container} className='cg-Specimen-Html'>
-        {toggle}
+        {options.device ? responsive : null} {toggle}
         <div style={{...styles.content, ...exampleStyles}}>
-          {frame ? <Frame>{content}</Frame> : content }
+          {frame || options.device
+            ? <Frame width={options.device ? screenSize.width : parentWidth} parentWidth={parentWidth} height={options.device ? screenSize.height : void 0}>
```

Comment:
There's no need to set the `Frame` width when no device is selected (falls back to `100%` anyway), right?